src/components/DepositWidget/index.tsx

```diff
@@ -99,7 +272,16 @@ const DepositWidget: React.FC = () => {
           </thead>
           <tbody>
             {balances &&
-              balances.map(tokenBalances => <Row key={tokenBalances.addressMainnet} tokenBalances={tokenBalances} />)}
+              balances.map(tokenBalances => (
+                <Row
+                  key={tokenBalances.addressMainnet}
+                  tokenBalances={tokenBalances}
+                  onEnableToken={(): Promise<void> => _enableToken(tokenBalances.address)}
+                  onSubmitDeposit={(balance): Promise<void> => _deposit(balance, tokenBalances.address)}
+                  onSubmitWithdraw={(balance): Promise<void> => _requestWithdraw(balance, tokenBalances.address)}
+                  onClaim={(): Promise<void> => _claim(tokenBalances.address)}
```

Comment:
There are a lot of functions in this one component, 280 lines already. Maybe an extracted custom hook, like
```
const create createFuncs = useCreateRowFuncs(..., [tokenBalances, whatever])

...
balances.map(tokenBalances => <Row key={...} {...createFuncs(tokenBalances.address)/>)
```